tests/functional/modules/test_zos_copy_func.py

```diff
@@ -1960,8 +1966,9 @@ def test_ensure_copy_file_does_not_change_permission_on_dest(ansible_zos_module,
 @pytest.mark.seq
 @pytest.mark.parametrize("ds_type", ["PDS", "PDSE", "SEQ"])
 def test_copy_dest_lock(ansible_zos_module, ds_type):
-    DATASET_1 = "USER.PRIVATE.TESTDS"
-    DATASET_2 = "ADMI.PRIVATE.TESTDS"
+    hosts = ansible_zos_module
+    DATASET_1 = get_dataset(hosts)
+    DATASET_2 = get_dataset(hosts)
```

Comment:
Same comment about lowercase, I think I will stop commenting this and just let you change that in other test cases.